src/workingWithData.ts

```diff
@@ -1,33 +1,48 @@
-import { v4 as uuid } from 'uuid'
 import { resolve } from 'path'
 import { readFile, open, writeFile, unlink, access, mkdir } from 'fs/promises'
+import { UserInfoType } from './types/userType'
+import { v4 as uuid } from 'uuid'
+import { createDataRequest } from './types/dataRequest'
 
-import { UserInfoType } from './types/user_type'
-
-export async function createFolderIfNotExist(path: string) {
+export async function createFolderIfNotExists(path: string): Promise<void> {
   const pathToDIr: string = resolve(path)
   try {
     await access(pathToDIr)
+    console.log('The dir is almost exists')
   } catch (error) {
     await mkdir(pathToDIr)
     console.log('The dir has been created')
   }
 }
 
+export async function createFileIfNotExists(
+  path: string,
+  fileName: string
+): Promise<void> {
+  const takePath: string = resolve(path, fileName)
+  try {
+    await access(takePath)
+    console.log('The file is almost exists')
+  } catch (error) {
+    await open(takePath, 'w+')
+    console.log('File has been created')
+  }
+}
+
 export async function createContent(
   path: string,
   fileName: string,
-  content: UserInfoType
+  content: string
 ): Promise<void> {
   const takePath: string = resolve(path, fileName)
 
   try {
-    await access(takePath)
-    await writeFile(takePath, JSON.stringify(content))
+    //  await writeFile(takePath, JSON.stringify(content), { flag: 'a' })
+
+    await writeFile(takePath, Object(content))
     console.log('Content has been added')
   } catch (error) {
-    await open(takePath, 'a+')
-    console.log('File was created')
```

Comment:
If your file doesn't exists, what will you do?

Reply:
Done